examples/src/models/chat/integration_googlevertexai-tools.ts

```diff
@@ -25,7 +25,7 @@ const geminiCalculatorTool: GeminiTool = {
 
 const model = new ChatVertexAI({
   temperature: 0.7,
-  model: "gemini-1.0-pro",
+  model: "gemini-pro-vision",
```

Comment:
Suggest "gemini-1.5-flash" or "gemini-1.5-flash-001" instead. (Pro Vision has been deprecated on the AI Studio side and will be supported for less time than 1.5 flash on the Vertex AI side.)